kubel.el

```diff
@@ -954,8 +954,9 @@ See https://github.com/kubernetes/kubernetes/issues/27081"
 REPLICAS is the number of desired replicas."
   (interactive (list (read-number "Replicas: ")))
   (if (kubel--is-scalable)
-      (let ((resource (kubel--get-resource-under-cursor)))
-        (kubel--exec (list "scale" kubel-resource resource "--replicas" (number-to-string replicas))))
+      (let* ((resource (kubel--get-resource-under-cursor))
+             (process-name (format "kubel:scale:%s/%s" kubel-resource resource)))
+        (kubel--exec process-name (list "scale" kubel-resource resource "--replicas" (number-to-string replicas))))
```

Comment:
Thanks for the fix! sorry about that, I'm running a different kubel version on my machine... I'll make sure next time to double test the branch that I use to send the upstream changes.